src/drafts/MarkdownEditor/MarkdownEditor.test.tsx

```diff
@@ -57,6 +57,8 @@ const render = async (ui: React.ReactElement) => {
 
   const queryForToolbarButton = (label: string) => within(getToolbar()).queryByRole('button', {name: label})
 
+  const getDefaultFooterButton = () => within(getFooter()).getByRole('link', {name: 'Markdown documentation'})
```

Comment:
minor suggestion, we could add some test coverage here to assert that default footer button appears when not using a custom footer